app/models/katalyst/navigation/item.rb

```diff
@@ -24,6 +28,22 @@ def layout?
         is_a? Layout
       end
 
+      # Entrypoint for combining additional options based on attribute rules
+      def item_options
+        options_for_target
+      end
+
+      def options_for_target
+        return {} if target == "self"
+
+        options = { target: target }
+
+        unless target == "_blank" || target == "_top"
+          options = { data: { turbo: true, turbo_frame: target } }
+        end
+        options
+      end
```

Comment:
(opt) case statement might be more readable

Reply:
if we were to add new targets (more turbo frames etc) we wont have to update this code
and have the special cases `_top` etc mapped out